src/Directory.Build.props

```diff
@@ -2,9 +2,22 @@
     <PropertyGroup>
         <PackageProjectUrl>http://fable.io</PackageProjectUrl>
         <RepositoryUrl>https://github.com/fable-compiler/Fable.git</RepositoryUrl>
-        <PackageLicense>https://raw.githubusercontent.com/fable-compiler/Fable/master/LICENSE</PackageLicense>
-        <PackageIconUrl>https://raw.githubusercontent.com/fable-compiler/Fable/master/docs/img/fable_logo.png</PackageIconUrl>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
@MangelMaxime would you prefer the actual license file here as well?

Reply:
It is true that I prefer the licence file in general.

My idea, is that using the licence file means that if a user read the code from Github or NuGet he will see the same source of truth ==> The same LICENSE.txt file

Related to one of this https://github.com/fable-compiler/fable-browser/pull/62 @nojaf sent

Reply:
Done.